backend/Functions/Edna.AssignmentLearnContent/Edna.AssignmentLearnContent/Properties/launchSettings.json

```diff
@@ -1,3 +1,7 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
 {

```

Comment:
lets add an empty line here

Reply:
An empty line was not increasing any readability in this file, instead it was seeming a bit more like noise. Which is why I left it as is.